.github/workflows/lint-golang.yml

```diff
@@ -53,7 +55,8 @@ jobs:
         git diff --exit-code -- go.mod go.sum
 
     - name: Check goreleaser configuration file
-      uses: goreleaser/goreleaser-action@v5
+      uses: goreleaser/goreleaser-action@v6
       with:
-        version: v1.24.0
-        args: check src/go/.goreleaser.yaml
+        distribution: goreleaser-pro
```

Comment:
It's moderately interesting that config check operation running with goreleaser pro binary is running fine without `GORELEASER_KEY`!